jujugui/static/gui/src/app/components/profile/model-list/model-list.js

```diff
@@ -80,44 +82,53 @@ class ProfileModelList extends React.Component {
   */
   _confirmDestroy(modelUUID) {
     this.setState({notification: null});
-    this.props.destroyModel(modelUUID, (errors, data) => {
-      if (errors) {
-        errors.forEach(error => {
-          this.props.addNotification({
-            title: 'Error destroying model',
-            message: error,
-            level: 'error'
+    this.props.destroyModel(
```

Comment:
yeah lets not reformat code for fun in a 6000line diff

Reply:
I do not in any way find formatting fun, I find it an absolute head melt.